wiki/entry_ehret.qmd

```diff
@@ -0,0 +1,34 @@
+---
+title: "Wiki Entry Template"
+subtitle: "BA New Media / Online Communication KF/BF A: Spring Term 2023"
+author: "Maren Ehret"
+categories:
+  - "Media Psychology"
+  - "Digital Media"
+  - "Well Being"
+  - "Digital Healthcare"
+  - Paper: Schmietow, B. & Marckmann, G. (2019). Mobile health ethics and the expanding role of autonomy. Medicine, health care, and philosophy, 22(4),               623–630. 
+---
+
+Key Quote: "Autonomous, empowered individuals taking care proactively of their health are pictured as the creators of this vision for future 
+healthcare, and may at the same time redefne their role in relation to the traditional, expert-driven healthcare system." (Schmietow & Marckmann, 2019)
+
+Main Argument: The article by Schmietow and Marckmann is part of a larger project in the debate about ethical issues of digital healthcare and focuses 
+on the role of autonomy regarding new digital health applications. With the growing of various mobile health (mhealth) technologies, a lot of ethical questions as well as questions about the user/patient's autonomy continue to arise. Schmietow and Marckmann argue that mhealth strengthens and empowers
+their independancy in making decisions about their own health. They continue to explain how more autonomy and self-determination can benefit and support
+self-care and finally create a form of democratised medecine. However, they do highlight the imortance of digital and health literacy.
+
+Relationship: Unfortunatley this research area, espacially the role of autonomy in mheath, is still very vague. However Meskó et al. explain the importance
```

Comment:
What do you mean when you say that it is vague? How could it be made more explicit?